### DISCUSSION TITLE
Organization Completely Missing from Code Search

### DISCUSSION THREAD

**Discussion by ahatzz11** [community/community#34495] [Other Feature Feedback, Questions, & Ideas] [answered]
👋  I am part of an organization - `FigureTechnologies` - that has both public and private repositories. We have successfully used codesearch for both types of repositories, but today we found that all of our searches returned 0 results. The FAQ led me here to create a discussion.

<img width="1256" alt="image" src="https://user-images.githubusercontent.com/6256032/192891052-a8d451bb-abc2-4119-b1f7-b0a235dd98ea.png">

Is there anything that we can do to ensure that code search works for us again? Many of the engineers in our organization use this preview regularly.

**Comment by mattt-seek** [comment]
Same here but an experiment that worked... a very specific search in 'All repos' is still finding things within our org.

**Reply by thieman** [reply]
I also have an example of this:

This turns up hits on public repos in the DataDog org: https://cs.github.com/?scopeName=All+repos&scope=&q=%22LEGACY_DATADOG_URLS%22

But adding the `org:DataDog` filter results in no hits: https://cs.github.com/?scopeName=All+repos&scope=&q=%22LEGACY_DATADOG_URLS%22+org%3ADataDog

**Comment by tvpham11** [comment]
Same here. Code search has worked (quite) well when limiting scope to the private org that I'm part of, but the feature seemed to go down at some point yesterday afternoon (Eastern Time).

**Comment by thieman** [comment]
Seems to be working again, anyone else still having problems?

**Reply by tvpham11** [reply]
Works for me. Thanks for the heads-up @thieman!

**Reply by ahatzz11** [reply]
@thieman We've had users in our organization reporting it being back online for them and I'm able to use it as well, I'm glad this was just a blip in the preview and not our entire org being removed.

**Comment by ahatzz11** [comment, marked as answer]
As of ~11:30am CT code search seems to be working again for me and the users of our organization. Others outside of my org are also reporting it working: 

- https://github.com/orgs/community/discussions/34495#discussioncomment-3765069
- https://github.com/orgs/community/discussions/34495#discussioncomment-3765197

Seems like this was just a blip in the preview, I'm just glad it's back 🎉

**Comment by tclem** [comment]
Apologies for the breakage here, this was a bug impacting organizations (and users) with uppercase characters. It should be fixed now!

**Reply by mattt-seek** [reply]
Thanks for the info.  Still _loving_ the new search.
